src/app.module.ts

```diff
@@ -1,17 +1,30 @@
 import { getNodeEnv } from '@fullstacksjs/toolbox';
 import { Module } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
+import Joi from 'joi';
 import path from 'path';
 
-import configuration from './config/configuration';
+import { PrismaModule } from './prisma/prisma.module';
 
 @Module({
   imports: [
     ConfigModule.forRoot({
       cache: false,
+      isGlobal: true,
       envFilePath: [path.join(process.cwd(), `.${getNodeEnv()}.env`)],
```

Comment:
I'm afraid I have to disagree with this approach.
Keeping the environment outside of the codebase is a better practice.

Reply:
what's your suggestion?

Reply:
Let's discuss on a call